Dockerfile

```diff
@@ -53,4 +52,6 @@ RUN echo "gem: --no-ri --no-rdoc --no-document" > /root/.gemrc
 
 COPY . /build_scripts
 
+RUN gem install bundler -v $(ruby -e "require 'yaml'; puts YAML.load_file('/build_scripts/config/options.yml')['bundler_version']")
```

Comment:
Also this doesn't honor the overrides that can be placed in the OPTIONS directory.